README.md

```diff
@@ -4,7 +4,14 @@ TODO: Describe.
 
 ## Usage
 
-TODO: Describe.
+You can provide the desired stop IDs to display either with the `stopIds` query parameter in the url...
+```
+localhost:5173/?stopIds=64,117
+```
+...or by setting the `VITE_STOP_IDS` environment variable when running the development server.
+```sh
+VITE_STOP_IDS=64,117 npm run dev
+```
```

Comment:
I think I'd like to revert this for now - this section will be for "end users" (thus shouldn't mention the development stuff) and we have to do an in-depth write up closer to deployment anyways (and we can worry about this then)